src/dipali/Assignment19.java

```diff
@@ -0,0 +1,33 @@
+package dipali;
+
+public class Assignment19 {
+	
+	public String getDetails(String input) {
+		int asciiUpper=0;
+		int asciilower=0;
+		if(input!=null) {
+			int len=input.length();
+			
+			int ascii=0;
+			for(int cnt=0;cnt<len;cnt++) {
+				ascii =ascii+(int) input.charAt(cnt);
```

Comment:
This is wrong approach ,
if(Character.isDigit(ch)){
      digit++;
}else if(Character.isLetter(ch)){
      letter++;
}